openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1168,9 +1170,21 @@ A Credential Issuer is identified by a case sensitive URL using the `https` sche
 
 The Credential Issuer's configuration can be retrieved using the Credential Issuer Identifier.
 
-Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`. 
+Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`.
+
+Communication with the Credential Issuer Metadata Endpoint MUST utilize TLS.
+
+To fetch the Credential Issuer Metadata, a requester MUST send a HTTP request using the GET method and the path formed following the steps above. The URL SHOULD NOT contain any query parameters. The Credential Issuer MUST return a JSON document compliant with this specification using the `application/json` media type and a HTTP Status Code 200.
+
+The Wallet is RECOMMENDED to send an `Accept-Language` Header in the HTTP GET request to indicate the particular language(s) it is interested in. The Credential Issuer is RECOMMENDED to send a subset of internationalized display data containing the requested language(s) and indicate the language(s) returned using the HTTP `Content-Language` Header. The requested language(s) MUST use the values defined in [@!RFC3066]. The Credential Issuer MAY ignore the `Accept-Language` Header and send all supported languages or any chosen default subset.
```

Comment:
```suggestion
The Wallet is RECOMMENDED to send an `Accept-Language` Header in the HTTP GET request to indicate the language(s) preferred for display. It is up to the Credential Issuer whether to

* send a subset the metadata containing internationalized display data for one or all of the requested languages and indicate returned languages using the HTTP `Content-Language` Header
* ignore the `Accept-Language` Header and send all supported languages or any chosen default subset.
```

Reply:
I like those changes! That is a lot clearer for implementers imho.

Reply:
I like it as well. Shall we keep the RFC reference?

Reply:
I'm good with the proposed language, thanks Kristina!

Reply:
@paulbastian can you please commit suggestion if you agree and request re-review from Joseph so that we can merge this PR during tomorrow's call?

Reply:
for the RFC reference, I think it applies to both Headers, so I added a sentence saying so?